airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -298,45 +299,27 @@ def _collect_bulk_errors(self, response: requests.Response) -> List[Optional[dic
             )
 
     def _job_healthcheck(self, response: requests.Response) -> Optional[Exception]:
-        try:
-            # save the latest request to retry
-            self._save_latest_request(response)
-
-            # get the errors, if occured
-            errors = self._collect_bulk_errors(response)
-
-            # when the concurrent job takes place,
-            # another job could not be created
-            # we typically need to wait and retry, but no longer than 10 min.
-            if self._has_running_concurrent_job(errors):
-                return self._job_retry_on_concurrency()
-
-            # when the job was already created and the error appears in the middle
-            if self._job_state and errors:
-                self._on_job_with_errors(errors)
-
-            # when the job was not created because of some errors
-            if not self._job_state and errors:
-                self._on_non_handable_job_error(errors)
+        errors = self._collect_bulk_errors(response)
 
-        except (ShopifyBulkExceptions.BulkJobBadResponse, ShopifyBulkExceptions.BulkJobError) as e:
-            raise e
-
-    def _job_send_state_request(self) -> requests.Response:
-        with self.session as job_state_request:
-            status_args = self._job_get_request_args(ShopifyBulkTemplates.status)
-            self._request = requests.Request(**status_args, auth=self.session.auth).prepare()
-            return job_state_request.send(self._request)
+        if self._job_state and errors:
+            self._on_job_with_errors(errors)
 
     def _job_track_running(self) -> None:
-        job_state_response = self._job_send_state_request()
-        self._job_healthcheck(job_state_response)
-        self._job_update_state(job_state_response)
-        self._job_state_to_fn_map.get(self._job_state)(response=job_state_response)
+        _, response = self._http_client.send_request(
+            http_method="POST",
+            url=self.base_url,
+            data=ShopifyBulkTemplates.status(self._job_id),
+            headers={"Content-Type": "application/graphql"},
+            request_kwargs={},
+        )
+        self._job_healthcheck(response)
+
+        self._job_update_state(response)
+        self._job_state_to_fn_map.get(self._job_state)(response=response)
 
     def _has_running_concurrent_job(self, errors: Optional[Iterable[Mapping[str, Any]]] = None) -> bool:
         """
-        When concurent BULK Job is already running for the same SHOP we receive:
+        When concurrent BULK Job is already running for the same SHOP we receive:
```

Comment:
Why would we change the `concurrent` to be `concurent` ? I believe there is an English word `concurrent` - [article](https://dictionary.cambridge.org/dictionary/english/concurrent)